exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/otlp/traces/SpanMarshaler.java

```diff
@@ -37,6 +37,7 @@ final class SpanMarshaler extends MarshalerWithSize {
   private final SpanLinkMarshaler[] spanLinkMarshalers;
   private final int droppedLinksCount;
   private final SpanStatusMarshaler spanStatusMarshaler;
+  private final int flags;
```

Comment:
Can we add a unit test?

Reply:
Sure. I added one but it's pretty cheesy....hopefully ok for now (better than nothing, which is what it was before).

Reply:
There was a unit test before. See `TraceRequestMarshalerTest`.